dev-libs/gtest/gtest-1.9.0~git.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="C++ test framework"
+DESCRIPTION="C++ testing utility
+Features:
+* An xUnit test framework
+* Test discovery
+* A rich set of assertions
+* User-defined assertions
+* Death tests
+* Fatal and non-fatal failures
+* Value-parameterized tests
+* Type-parameterized tests
+* Various options for running the tests
+* XML test report generation"
+HOMEPAGE="https://github.com/google/googletest"
+COPYRIGHT="2008-2016 Google Test developers"
+LICENSE="BSD (3-clause)"
+srcGitRev="ea31cb15f0c2ab9f5f5b18e82311eb522989d747"
+REVISION="1"
+SOURCE_URI="https://github.com/google/googletest/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="41a782bacfed269409e533599f2272910e6c7ccd0c3d03f9694c693916070180"
+SOURCE_FILENAME="googletest-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="googletest-$srcGitRev"
+PATCHES="gtest-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gtest$secondaryArchSuffix = $portVersion
+	lib:libgmock$secondaryArchSuffix
+	lib:libgmock_main$secondaryArchSuffix
+	lib:libgtest$secondaryArchSuffix
+	lib:libgtest_main$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	gtest${secondaryArchSuffix}_devel = $portVersion
+	devel:libgmock$secondaryArchSuffix
+	devel:libgmock_main$secondaryArchSuffix
+	devel:libgtest$secondaryArchSuffix
+	devel:libgtest_main$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	gtest$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python2
```

Comment:
@Begasus, in case you would like to revert to python3:
```diff
-	cmd:python2
+	cmd:python3
```
I think this would work, thanks to commit 298e4d49689975287246a3bf99592a169f8f29d8 by @pulkomandy.

Reply:
Ok, I reverted it since there wasn't any response on whether to use 2 or 3 ...